sklearn/multiclass.py

```diff
@@ -637,3 +649,481 @@ def predict(self, X):
         Y = np.array([_predict_binary(e, X) for e in self.estimators_]).T
         pred = euclidean_distances(Y, self.code_book_).argmin(axis=1)
         return self.classes_[pred]
+
+
+def _binomialCoeff(n, k):
```

Comment:
You probably want something like this:
http://docs.scipy.org/doc/scipy-0.16.0/reference/generated/scipy.misc.comb.html